typescript/widgets/src/components/SearchMenu.tsx

```diff
@@ -91,12 +99,28 @@ export function SearchMenu<
     [data, searchQuery, sortState, filterState, searchFn],
   );
 
+  const handleKeyDown = useCallback(
```

Comment:
This approach is okay but what I expected to see was that the input field get wrapped by a `form`. That way, the enter key would automatically trigger the form submit, and then you'd execute your callback here in that submit fn. WDYT of that approach? Feels a bit more robust to me than watching for a specific key name maybe?

Reply:
Wrapped the `input` around a `form` and change `handleKeyDown` to `handleSubmit`